index.html

```diff
@@ -0,0 +1,346 @@
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Project1</title>
+  </head>
+  <body class="bg-secondary">
+    <div class="container-fluid">
+      <!-- logo & hamburger menu -->
+      <header class="logo-menubar d-flex1">
+        <img src="./images/logo.svg" alt="logo" />
+        <img
+          class="menu-icon"
+          src="./images/menu.svg"
+          alt=""
+        />
+      </header>
+      <!-- end of logo & hamburger menu -->
+      <!-- main body start -->
+      <main class="main display-flex2">
+        <div class="intro">
+          <h1>
+            I’m Balqees Ibrahim Glad to see you!
+          </h1>
+          <p>
+            I’m a software developer! I can help
+            you build a product feature or
+            website. Look through some of my work
+            and experience! If you like what you
+            see and have a project you need coded,
+            don’t hesitate to contact me.
+          </p>
+        </div>
+        <a class="connect" href="#"
+          >let's connect</a
+        >
+        <div class="social-icons">
+          <a href="https://twitter.com/qeescode">
+            <img
+              src="./images/twitter.svg"
+              alt=""
+            />
+          </a>
+          <a
+            href="https://www.linkedin.com/in/balqees-ibrahim-911a81204/"
+            class="p1"
+          >
+            <img
+              src="./images/linkedin.svg"
+              alt=""
+            />
+          </a>
+          <a href="#" class="p1">
+            <img
+              src="./images/medium.svg"
+              alt=""
+            />
+          </a>
+          <a
+            href="https://github.com/balqees0325"
+            class="p1"
+          >
+            <img
+              src="./images/github.svg"
+              alt=""
+            />
+          </a>
+          <a href="#" class="p1">
+            <img src="./images/hand.svg" alt="" />
+          </a>
+        </div>
+      </main>
+    </div>
+    <!-- end of first container -->
+    <!-- beginning of section -->
+    <article>
+      <section class="section2 d-flex2">
+        <img
+          src="./images/Snapshoot-Portfolio.svg"
+          alt=""
+          class="showcase-img"
+        />
+        <h2>Tonic</h2>
+        <div class="article">
+          <p class="canopy">CANOPY</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>Back End DEv</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>2015</p>
+        </div>
+        <p>
+          A daily selection of privately
+          personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="available-articles">
+          <a href="#">html</a>
+          <a href="#">css</a>
+          <a href="#">javaScript</a>
```

Comment:
- [ ] As mentioned above about lists, please use `ul` for all your lists with `li` for these technologies
Here you can use `ul` for the group of technologies with technologies such as html inside `li` tags
Please update them for all technologies in all works below also